src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -139,7 +66,10 @@ private boolean hasEventId(CaseTypeDefinition caseTypeDefinition, String eventId
     public void validateData(Map<String, JsonNode> data,
                              CaseTypeDefinition caseTypeDefinition,
                              final CaseDataContent content) {
-        caseTypeService.validateData(data, caseTypeDefinition);
-        validateOrganisationPolicy(caseTypeDefinition.getId(), content);
+        caseTypeService.validateData(createValidationContext(caseTypeDefinition.getId(), content, data, caseTypeDefinition));
+    }
+
+    public ValidationContext createValidationContext(String caseTypeId, CaseDataContent content, Map<String, JsonNode> data, CaseTypeDefinition caseTypeDefinition) {
```

Comment:
`String caseTypeId` is not needed, you are already passing `CaseTypeDefinition caseTypeDefinition` you can get it from there

Reply:
DONE